### DISCUSSION TITLE
Suggest adding support for Govee grill meat thermometers

### DISCUSSION THREAD

**Discussion by tmarkson** [custom-components/ble_monitor#490] [Ideas]
An idea for the distant future, supporting these devices for the baker, smoker, grill-ers!  They are similar to the environment sensors already supported but containing remote temperature probes to be placed on whatever is cooking. 

Double thermometer (H5183) https://www.amazon.com/gp/product/B092ZTD96V/
Single thermometer (H5182) https://www.amazon.com/gp/product/B094N2FX9P/

**Comment by Ernst79** [comment]
Sure, if you have one of these, let us know. We need some examples of the BLE advertisements to add these devices. The [FAQ](https://custom-components.github.io/ble_monitor/faq#my-sensor-is-not-in-the-list-of-supported-ones-how-to-request-implementation) gives instructions on how to get these.

**Comment by tmarkson** [comment]
I'm on to this, I do have one of the double units.  Thanks for the link. Forgot about this thread!

**Comment by Ernst79** [comment]
H5183 should already be supported.

**Reply by tmarkson** [reply]
Cool, I will try it.

**Reply by tmarkson** [reply]
I have updated the integration and left the thermometer powered on.  I didn't see a new device populated so I deleted the Passive BLE integration and set it up again.  Still there is not a new device, looking for the H5183.  

Do you have a suggestion?
